lib/DDH/UserPage/Generate.pm

```diff
@@ -105,6 +110,19 @@ sub generate {
             }
         );
         close $fh;
+
+        # JSON dump of user's data
+        my $json_build_dir = $self->json_dir( $contributor );
+        open my $jfh, '>:encoding(UTF-8)', "$json_build_dir/index.html" or die();
+        print $jfh $self->xslate->render(
```

Comment:
@jbarrett Done, but for some reason it doesn't get nicely formatted like when I was using a template :(